app/views/pages/index.html.erb

```diff
@@ -10,3 +10,18 @@
     <%= link_to "Subscribe to newsletter", new_subscription_path %>
   </nav>
 <% end %>
+
+
+<%= form_tag do %>
+  <%= label_tag :email %>
+  <%= text_field_tag :email, '',
+        inputmode: :email,
+        list: :emails,
```

Comment:
Here, `email_field_tag` generates HTML looking like:

```html
<input type="email" value="" list="emails" data-remote="true" data-method="get" data-url="/emails" oninput="jQuery(this).change();">
```
The `list="emails"` attribute refers to the `<datalist id="emails">` element below.